### PR TITLE
#2335 Rephrase mute options in help text

### PR DESCRIPTION
**Pull request by pljones**
**Short description of changes**

Rephrases the mute options in help text

CHANGELOG: Rephrase mute options in help text

**Context:**
Fixes: #2335

**Does this change need documentation? What needs to be documented and how?**

Website help must be in line with application help.

**Status of this Pull Request**

Tested locally.

**What is missing until this pull request can be merged?**

Sign off

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1094,8 +1094,8 @@ QString UsageArguments ( char** argv )
            "Client only:\n"
            "  -c, --connect         connect to given Server address on startup\n"
            "  -j, --nojackconnect   disable auto JACK connections\n"
-           "  -M, --mutestream      starts the application in muted state\n"
-           "      --mutemyown       mute me in my personal mix (headless only)\n"
+           "  -M, --mutestream      prevent others on a server from hearing what I play\n"
+           "      --mutemyown       prevent me from hearing what I play in the server mix\n"
```
Comment (ann0see) [inline src/main.cpp:1098]: Does it still work headless only?

Comment (pljones) [reply src/main.cpp:1098]: Oh yes, I forgot about than... (mindless c-n-p...).

I have it on my long list of things to do to investigate how to make it work with the GUI.  It'll help the JSON-RPC interface, too.

Comment (pljones) [reply src/main.cpp]: OK, updated and I also updated the "confirmation text" the program issues during start up.

**Review by ann0see — COMMENTED** [review]
Looks good!
Waiting on @gilgongo for a final word.

**Review by ann0see — APPROVED** [review]

**Review by hoffie — APPROVED** [review]

**Pull request merged**